docs/content/en/docs/reference/clusterspec/optional/cni.md

```diff
@@ -134,7 +134,46 @@ EKS Anywhere will create the required NetworkPolicy objects for its core compone
 2. Switching from `always` mode: When switching from `always` to `default` mode, EKS Anywhere
 will not delete any of the existing NetworkPolicy objects, including the ones required
    for EKS Anywhere components (listed above). The user must delete NetworkPolicy objects as needed.
-   
+
+### Use a custom CNI
+
+EKS-A can be configured to skip EKS-A Cilium CNI upgrades via the `skipUpgrade` field. `skipUpgrade` 
+can be `true` or `false`. When not set, it defaults to `false`.
+
+When creating a new cluster with `skipUpgrade` enabled, EKS-A Cilium will be installed. When the 
+cluster successfully provisions, EKS-A Cilium may be uninstalled and replaced with a different CNI.
+Subsequent upgrades to the cluster will not attempt to upgrade or re-install EKS-A Cilium.
+
+Once enabled, `skipUpgrade` cannot be disabled.
+
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: my-cluster-name
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - 192.168.0.0/16
+    services:
+      cidrBlocks:
+      - 10.96.0.0/12
+    cniConfig:
+      cilium: 
+        skipUpgrade: true
+```
+
+{{% alert title="Warning" color="warning" %}}
+Clusters created using the Full Lifecycle Controller prior to v0.15 that have removed the EKS-A
+Cilium CNI must manually populate their `Cluster` object with the following annotation to ensure
```

Comment:
How do they manually do this? via kubectl -f apply with an updated cluster spec?

Reply:
There's multiple ways to do it. Do we need to prescribe how? We could be clearer about the object they're editing though given there are multiple Cluster objects.

Reply:
This sentence may not be clear to newer users. I'm assuming the annotation goes on the `Cluster` spec and I apply that via my choice of K8s API conformant tool (GitOps, Terraform, kubectl, etc.).

Reply:
Made clear its a cluster spec object and the exact object that needs annotating.